src/frontend/packages/cloud-foundry/src/cf-entity-generator.ts

```diff
@@ -653,7 +653,14 @@ function generateGitCommitEntity(endpointDefinition: StratosEndpointExtensionDef
     schema: cfEntityFactory(gitCommitEntityType),
     label: 'Git Commit',
     labelPlural: 'Git Commits',
-    endpoint: endpointDefinition
+    endpoint: endpointDefinition,
+    nonJetstreamRequest: true,
```

Comment:
Other git entities should also go through same process (`nonJetstreamRequest`, apply guid(?), etc)

Reply:
They should eventually, but I didn't get around to migrating them to the new api pipeline. They had their own effect that handled the GitHub related requests.

I've created https://github.com/cloudfoundry-incubator/stratos/issues/3969 to migrate the rest of the github requests.